README.md

```diff
@@ -776,10 +776,12 @@ IEEE Conf. Computational Photography (ICCP), 2015. [PDF](http://compphotolab.nor
 - <a name="Leroux18arxiv"></a>Leroux, T., Ieng, S.-H., Benosman, R.,  
 *[Event-Based Structured Light for Depth Reconstruction using Frequency Tagged Light Patterns](https://arxiv.org/pdf/1811.10771)*,  
 arXiv:1811.10771, 2018.
+- <a name="mangalore2020neuromorphic"></a>Mangalore, A. R., Seelamantula, C. S., & Thakur, C. S., 
```

Comment:
No & before Thakur,
The 2nd part of the key should be, if possible, lsp (to coincide with the abbreviation in the doi
Need two blank spaces at the end of the line to break it on github, so that the title of the paper starts in a new line. Please check the preview. Thank you

Reply:
I made a new commit. Is it okay now?

Reply:
yes